recipes/fastdtw/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "fastdtw" %}
+{% set version = "0.3.2" %}
+{% set sha256 = "09a24d62a8eb72f87c03692dffae4b8d8499b53f503e9abce5e717c5b705dff9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  # noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+    - Cython
+    - numpy
+    - setuptools
+    - six
+    - pytest-runner
+
+  run:
+    - python
+    - numpy
+    - six
+
+test:
+  imports:
+    - fastdtw
+
+about:
+  home: https://github.com/slaypni/fastdtw
+  license: MIT
```

Comment:
Please add the ``license_file``. I saw that the license is not being packaged in pypi, it would be good if you open a pull request for that in the project  :)

But for now you can add the license file manually, please see
https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Reply:
I didn't even notice. Thanks.